src/editor/index.tsx

```diff
@@ -22,20 +24,73 @@ export const EditorScreen = () => {
   const [request, setRequest] = useState('');
   const [echo, setEcho] = useState<JSX.Element>(<></>);
   const [requestResult, setRequestResult] = useState('');
-  const [enableABAC, setEnableABAC] = useLocalStorage(true, 'ENABLE_ABAC');
-
   const [customConfig, setCustomConfig] = useState('');
+  const [share, setShare] = useState('');
+
+  function setPolicyPersistent(text: string): void {
+    set(Persist.POLICY, text);
+    setPolicy(text);
+  }
+
+  function setModelTextPersistent(text: string): void {
+    set(Persist.MODEL, text);
+    setModelText(text);
+  }
+
+  function setCustomConfigPersistent(text: string): void {
+    set(Persist.CUSTOM_FUNCTION, text);
+    setCustomConfig(text);
+  }
+
+  function setRequestPersistent(text: string): void {
+    set(Persist.REQUEST, text);
+    setRequest(text);
+  }
+
+  useEffect(() => {
+    const hash = window.location.hash.slice(1);
+    if (hash) {
+      setEcho(<Echo>Loading Shared Content...</Echo>);
+      fetch(`https://dpaste.com/${hash}.txt`)
+        .then(resp => resp.text())
+        .then(content => {
+          const sharedContent = JSON.parse(content) as ShareFormat;
+          setPolicyPersistent(sharedContent.policy);
+          setModelTextPersistent(sharedContent.model);
+          setCustomConfigPersistent(sharedContent.customConfig);
+          setRequestPersistent(sharedContent.request);
+          setRequestResult('');
+          setEcho(<Echo>Shared Content Loaded.</Echo>);
```

Comment:
When you refresh this page, it will re-load the shared content and lost user data, so add an identifier to indicate that the shared data has been loaded to avoid multiple loads.